components/card.css

```diff
@@ -0,0 +1,95 @@
+.card,
+.card-horizontal {
+    border: 1px solid var(--theme300);
+    padding: 1rem;
+    line-height: 1.5;
+    text-align: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--theme50);
+    border-top-left-radius: 1rem;
+    border-bottom-right-radius: 1rem;
+    position: relative;
+    min-width: 15rem;
+}
+
+.card {
+    max-width: 20rem;
+    flex-direction: column;
+}
+
+.card-horizontal {
+    justify-content: space-between;
+    max-width:35rem;
+}
+
+.card-img {
+    width: 100%;
+    min-width: 15rem;
+    height:30vh;
+    border-radius: 1rem;
```

Comment:
Here you are using align self without display flex. Try adding display flex if you are using flex properties

Reply:
Thanks for previewing. I am using .card-img inside .card-horizontal so flex is added there